src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form_field {
+  margin-bottom: 20px;
+}
+
+.form_field:not(:last-child) {
+  margin-top: 20px;
+}

```

Comment:
You can leave only the margin-top for from fields

Reply:
done